packages/ui/src/utils/references.ts

```diff
@@ -0,0 +1,112 @@
+import { has } from './utils';
```

Comment:
Copied from style-dictionary@3.9.1. Only change is using our util `has` in favor of `hasOwnProperty`